### ISSUE TITLE
Proper Cruise control

### ISSUE THREAD

**Issue by Xerothis** [New-Chance-RP/Feedback#652]
<h2>What bug have you found?</h2>

So basically I've noticed that the new cruise control doesn't actually stop at your chosen speed, instead it keeps on speeding up, which in my opinion is kind of annoying since most of the time I would like to cruise around town at a steady speed, but I can't since it just keeps on accelerating, I think the old cruise control was better than the new one.

<h2>What do you expect should happen?</h2>

Basically the cruise control should stop at the desired speed, as it does in real life as well.

<h2>Anything to support your bug report?</h2>

-

**Comment by Senlarus** [comment]
Should be fixed.
https://discord.com/channels/680278765179502645/688123847047774210/785962816166428702
